watertap/unit_models/electrodialysis_0D.py

```diff
@@ -477,9 +581,15 @@ def build(self):
         if self.config.is_isothermal:
             self.diluate.add_isothermal_assumption()
         self.diluate.add_momentum_balances(
-            balance_type=self.config.momentum_balance_type, has_pressure_change=False
+            balance_type=self.config.momentum_balance_type,
+            has_pressure_change=self.config.has_pressure_change,
+        )
+        add_object_reference(
+            self, "dens_mass", self.diluate.properties_in[0].dens_mass_phase["Liq"]
```

Comment:
Maybe worth in-line commenting or documenting somewhere that this only works when `dens_mass` and `visc_d` are assumed constant at all `length_domain` elements for the diluate and concentrate channels (does not create separate for each channel)

Reply:
Thanks! I added in-line comments showing that den_mass and visc_d  in concentrate and diluate channels are the same. As for the length_domain, I discussed with Xiangyu, we don't think changes are needed since all variables in 0D are not length-indexed.